blackbox/docs/sql/joins.txt

```diff
@@ -76,6 +76,65 @@ the other table::
     +----+------------+------------------+
     SELECT 4 rows in set (... sec)
 
+Left Joins
+----------
+
+**Left** join returns tuples of the *left* relation matching tuples of the *right*
+relation and dangling tuples of the *right* relation padded with null values::
+
+    cr> select a.name as article_name, uv.name as user_visited, uv.visits
+    ... from articles a left join uservisits uv
+    ... on a.id = uv.id
+    ... order by a.id;
+    +-------------------------------+--------------+--------+
+    | article_name                  | user_visited | visits |
+    +-------------------------------+--------------+--------+
+    | Starship Titanic              | Trillian     |      3 |
+    | Towel                         |              |        |
+    | Kill-o-Zap blaster pistol     |              |        |
+    | Infinite Improbability Drive  |              |        |
+    +-------------------------------+--------------+--------+
+    SELECT 4 rows in set (... sec)
+
+Right Joins
+-----------
+
+**Right** join returns tuples of the *right* relation matching tuples of the *left*
+relation and dangling tuples of the *left* relation padded with null values::
+
+    cr> select a.name as article_name, uv.name as user_visited, uv.visits
```

Comment:
not sure if this is a good example. User visiting articles? Think it would be good if we used a more common case as example